src/mechanics/constitutive/damageLaws/DamageLawLinear.h

```diff
@@ -1,54 +1,54 @@
 #pragma once
 
-#include <memory>
+#include <algorithm> // std::min
 #include "mechanics/constitutive/damageLaws/DamageLaw.h"
 
 namespace NuTo
 {
 namespace Constitutive
 {
-
+//! @brief linear damage law
+//! Peerlings et al.. Gradient enhanced damage quasi-brittle materials 1996.
```

Comment:
Personally, I'd like to find a link to the paper in the doxygen documentation, maybe like this:
~~~cpp
//! DOI: [10.1002/(SICI)1097-0207(19961015)39:19<3391::AID-NME7>3.0.CO;2-D](http://dx.doi.org/10.1002/(SICI)1097-0207(19961015)39:19<3391::AID-NME7>3.0.CO;2-D)
~~~